net-dns/dnsviz/dnsviz-0.8.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
```

Comment:
does not it work with 3.7?

Reply:
It seems upstream hasn't been tested dnsviz on 3.7 yet.I will ask cdeccio(author).If it is ok i will add 3.7.

Currently Dependencies

    python (2.7/3.4/3.5/3.6)

Reply:
I tested it (by running repoman), dev-python/libnacl has not been enabled with python3.7 yet. You should get this one merged first, then study (and implement) python3.7 support for libnacl (new PR), and make a new PR to enable python3.7 with this. It's quite simple as updating PYTHON_COMPAT doesn't need a revbump.

I see upstream enables python tests for this package, and seems to have some "offline_tests.py" possibility. Could you implement that in this ebuild?

Reply:
Sure i can implement 3.7.
I am working on that now.

*I will make new PR for libnacl and test it. 
*Then update this PR  with 3.7 support, if tests is ok.

Reply:
It seems ok.

**libnacl**

```
python3.7 runtests.py
test_gcm_aead (test_aead.TestAEAD) ... skipped 'AES256-GCM AEAD not available'
test_ietf_aead (test_aead.TestAEAD) ... ok
test_auth_verify (test_auth_verify.TestAuthVerify) ... ok
test_onetimeauth_verify (test_auth_verify.TestAuthVerify) ... ok
test_key_blake (test_blake.TestBlake) ... ok
test_keyless_blake (test_blake.TestBlake) ... ok
test_publickey (test_dual.TestDual) ... ok
test_secretkey (test_dual.TestDual) ... ok
test_sign (test_dual.TestDual) ... ok
test_publickey (test_public.TestPublic) ... ok
test_secretkey (test_public.TestPublic) ... ok
test_secret_box (test_raw_auth_sym.TestSecretBox) ... ok
test_key_generichash (test_raw_generichash.TestGenericHash) ... ok
test_keyless_generichash (test_raw_generichash.TestGenericHash) ... ok
test_hash (test_raw_hash.TestHash) ... ok
test_box (test_raw_public.TestPublic) ... ok
test_box_seal (test_raw_public.TestPublic) ... ok
test_boxnm (test_raw_public.TestPublic) ... ok
test_gen (test_raw_public.TestPublic) ... ok
test_randombytes (test_raw_random.TestRandomBytes)
copied from libsodium default/randombytes.c ... ok
test_randombytes_random (test_raw_random.TestRandomBytes) ... ok
test_randombytes_uniform (test_raw_random.TestRandomBytes) ... ok
test_secretbox (test_raw_secret.TestSecret) ... ok
test_box (test_raw_sign.TestSign) ... ok
test_gen (test_raw_sign.TestSign) ... ok
test_save_load (test_save.TestSave) ... ok
test_save_load_secret (test_save.TestSave) ... ok
test_save_load_sign (test_save.TestSave) ... ok
test_save_perms (test_save.TestSave) ... ok
test_publickey_only (test_seal.TestSealed) ... ok
test_secretkey (test_seal.TestSealed) ... ok
test_secret (test_secret.TestSecret) ... ok
test_sign (test_sign.TestSigning) ... ok
test_verify16 (test_verify.TestVerify) ... ok
test_verify32 (test_verify.TestVerify) ... ok
test_verify64 (test_verify.TestVerify) ... ok
test_different (test_verify.TestVerifyBytesEq) ... ok
test_different_length (test_verify.TestVerifyBytesEq) ... ok
test_equal (test_verify.TestVerifyBytesEq) ... ok
test_invalid_type (test_verify.TestVerifyBytesEq) ... ok
test_library_version_major (test_version.TestSodiumVersion) ... ok
test_library_version_minor (test_version.TestSodiumVersion) ... ok
test_version_string (test_version.TestSodiumVersion) ... ok

----------------------------------------------------------------------
Ran 43 tests in 0.115s

OK (skipped=1)
```

**dnsviz**

```
python3.7 offline_tests.py 
............
----------------------------------------------------------------------
Ran 12 tests in 6.697s

OK
```